app/lib/BaseModelWithAttributes.php

```diff
@@ -2199,7 +2199,10 @@ public function htmlFormElementForAttributeSearch($po_request, $pm_element_code_
 			$pa_options['format'] = $vs_format;
 			
 			if ($vs_rel_types = join(";", caGetOption('restrictToRelationshipTypes', $pa_options, array()))) { $vs_rel_types = "/{$vs_rel_types}"; }
-			
+
+			if (count($va_element_set) > 1) {		// multiple elements cannot use the same name
+				unset($pa_options['name']);
+			}
 			foreach($va_element_set as $va_element) {
 				$va_override_options = array();
 				if ($va_element['datatype'] == 0) {		// containers are not active form elements
```

Comment:
This is the check that prevents this issue when the parent element is a container.
Considered replicating with a check on if parent_id exists, but seems more likely to cause unintended side effects?

Reply:
Have you checked the effect on proper containers with children? Do the values passed in still work?

Reply:
This same value is already unset for containers on the next line